app/models/post.rb

```diff
@@ -49,6 +49,8 @@ class Post < ApplicationRecord
   belongs_to :spoiled_unit, polymorphic: true
   has_many :post_likes, dependent: :destroy
   has_many :comments, dependent: :destroy
+  # TODO: what happens if the original is destroyed?
```

Comment:
I think for now the "Edited at" indicator is enough to delay potential abuse, and eventually we'll have an edit history on posts too